src/sentry/utils/batching_kafka_consumer.py

```diff
@@ -215,6 +215,7 @@ def __init__(
         metrics_sample_rates=None,
         metrics_default_tags=None,
         commit_on_shutdown: bool = False,
+        **_kwargs,
```

Comment:
why is this necessary?

Reply:
The v1 and v2 consumers have different options which will just be passed down further as kwargs, in order to avoid errors because of this, this just accepts anything that was passed from higher up.
I prefer that to explicitly `pop`-ing off any additional argument, as that will break when you forget to do so.